### DISCUSSION TITLE
Allow us to choose how the collections are organized.

### DISCUSSION THREAD

**Discussion by sarahtharp** [Kong/insomnia#8032] [Ideas] [answered]
It is extremely annoying that I can't just set the collections in alphabetical order and have them stay there. Constantly moving the "recently used" collection to the top is NOT helpful when I already know which collection I'm looking for. PLEASE STOP MOVING THEM AROUND.

**Comment by subnetmarco** [comment, marked as answer]
Hi Sarah, you can organize your collections in many ways, including alphabetical order:

![Screenshot 2024-10-02 at 9 23 36 AM](https://github.com/user-attachments/assets/ae449caf-5a7e-43cb-8071-2e473e682b45)
